api/core/workflow/nodes/agent/agent_node.py

```diff
@@ -348,3 +353,10 @@ def _fetch_model(self, value: dict[str, Any]) -> tuple[ModelInstance, AIModelEnt
         )
         model_schema = model_type_instance.get_model_schema(model_name, model_credentials)
         return model_instance, model_schema
+
+    def _remove_unsupported_model_features_for_old_version(self, model_schema: AIModelEntity) -> AIModelEntity:
+        if model_schema.features:
+            for feature in model_schema.features:
+                if feature.value not in AgentOldVersionModelFeatures:
```

Comment:
Here, feature is an enum type, and feature.value is of string type.

The code snippet:

```
feature.value not in AgentOldVersionModelFeatures,
```
results in the following error: TypeError: unsupported operand type(s) for 'in': 'str' and 'EnumType'.

Reply:
This issue occurs in Python 3.11